entities/user.entities.js

```diff
@@ -12,10 +12,10 @@ const UserSchema = new Schema({
   email: { type: String, required: true },
   slackId: { type: String },
   role: { type: String, default: 0, index: true },
-  encryptedPassword: { type: String, required: true },
+  encryptedPassword: { type: String },
```

Comment:
why no longer required?

Reply:
Because when I generate a user with the slackData he doesn't have a password and I can't save him.